[Warning] JavaDocs warning in kernel core

The following is printed when trying to build carbon-kernel:

[WARNING] /home/nira/wso2/git/carbon-kernel/core/src/main/java/org/wso2/carbon/kernel/internal/CarbonStartupHandler.java:39: warning: no @param for serverName
[WARNING] public static void logServerStartupTime(String serverName) {
[WARNING] ^
[WARNING] /home/nira/wso2/git/carbon-kernel/core/src/main/java/org/wso2/carbon/kernel/utils/manifest/ManifestElement.java:340: warning: no @param for bundle
[WARNING] public static List<ManifestElement> parseHeader(String header, String value, Bundle bundle)
[WARNING] ^

--- a/core/src/main/java/org/wso2/carbon/kernel/internal/CarbonStartupHandler.java
+++ b/core/src/main/java/org/wso2/carbon/kernel/internal/CarbonStartupHandler.java
@@ -35,6 +35,8 @@ public class CarbonStartupHandler {
 
     /**
      * Log the server start up time.
+     *
+     * @param serverName Server name to be in the log
      */
     public static void logServerStartupTime(String serverName) {
         double startTime = Long.parseLong(System.getProperty(org.wso2.carbon.kernel.Constants.START_TIME));

--- a/core/src/main/java/org/wso2/carbon/kernel/utils/manifest/ManifestElement.java
+++ b/core/src/main/java/org/wso2/carbon/kernel/utils/manifest/ManifestElement.java
@@ -332,6 +332,7 @@ public class ManifestElement {
      * @param header the header name to parse.  This is only specified to provide error messages
      *               when the header value is invalid.
      * @param value  the header value to parse.
+     * @param bundle OSGi bundle
      * @return the array of ManifestElements that are represented by the header value; null will be
      * returned if the value specified is null or if the value does not parse into
      * one or more ManifestElements.

